src/basic/env-file.c

```diff
@@ -79,7 +79,7 @@ static int parse_env_file_internal(
                                 n_key = 0;
                         } else if (c == '=') {
                                 state = PRE_VALUE;
-                                last_value_whitespace = SIZE_MAX;
```

Comment:
Sorry, but this change is definitely spurious. We're about to read the value here, hence the previous state for value gets reset.